app/uk/gov/hmrc/vatregisteredcompaniesapi/models/package.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompaniesapi
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDateTime, LocalTime}
+
+import scala.util.Random
+
+package object models {
+
+  type CompanyName = String
+  type VatNumber = String
+  type ConsultationNumber = String
+  type ProcessingDate = LocalDateTime
+
+  object ConsultationNumber {
+    def generate: ConsultationNumber =
+      new Random().alphanumeric.filter(x =>
+        x.toLower >= 'a' && x.toLower <= 'z'
+      ).take(9).toList.mkString
```

Comment:
this looks like it would generate lower and upper case characters - what was the requirement of the ID here? Also, it isn't a number as the name suggests

Reply:
The name comes from the project/user-interface - yes lower and upper was required.